internal/shards/repository.go

```diff
@@ -37,29 +38,48 @@ type sqlRepository struct {
 }
 
 func (r *sqlRepository) Lookup(shardKey string) (string, error) {
```

Comment:
Was there a bug found with these SQL calls?

Reply:
@adamdecaf no bugs, it was the `gosec` linter asking us to reimplement how we query

Reply:
e.g. ```Rows/Stmt was not closed (sqlclosecheck)
	rows, err := stmt.Query(
```

Reply:
Oh nice. I thought I had fixed those up already.